cmd/datamon/cmd/root.go

```diff
@@ -49,6 +49,19 @@ var logFatalln = log.Fatalln
 var logFatalf = log.Fatalf
 var osExit = os.Exit
 
+// infoLogger wraps informative messages to os.Stdout without cluttering expected output in tests.
+// To be used instead on fmt.Printf(os.Stdout, ...)
+var infoLogger = log.New(os.Stdout, "", 0)
+
+func wrapFatalln(msg string, err error) {
```

Comment:
can this replace `logFatalf` everywhere?

Reply:
@ransomw1c, well it could if `wrapFatalf` interprets a `nil` as `err` like `logFataln` does. I can do that if you prefer (and yes this would make it only 1 func to mock). Deal?

Reply:
to clarify, the motivating principle is that of consistency:  i'd prefer not to have two functions with the same semantic meaning of 

> report an error and exit

i'm confident that your choices about api design are reasonable.

Reply:
@ransomw1c ok will merge both funcs then